ws-nextjs-app/integration/pages/live/serbian/canonical.test.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @service serbian
+ * @pathname /serbian/live/media-23179005/lat
+ */
+
+import { runHeaderTests } from '../../../../../src/integration/common';
+
+describe('Canonical', () => {
+  describe(pageType, () => {
+    runHeaderTests();
```

Comment:
Can we add another test in here to assert that we can see the live page title (which we could derive from the page data? I am concerned that these integration tests are "passing", when in reality, I think a 404 page is being rendered. (The data is in the wrong place, so there would be a 404 data error).

I am not sure if it exists, but I thought there was logic to ensure that any page that is under integration test returns a 200 status (except for the error pages), but maybe that's only the cypress tests...?

Reply:
I think I've found it: we should run `runCommonCrossPlatformTests` instead of just the header tests (because this is only checking the header / nav bar, which will "pass" for error pages, rather than the content of the page). Can we do that before updating the serbian lat json file, and ensure that they fail?

Reply:
invoking `runCommonCrossPlatformTests` results in snapshot failures (as expected) when the data folder is routed with the `{id}.json` instead of `lat.json` so will update the file path now and re-run the snapshots to update:

<img width="1032" alt="image" src="https://github.com/user-attachments/assets/db73e0eb-bab8-4aeb-ac17-10dd7c9d7062">